src/main/java/io/blockfrost/sdk/api/model/ScriptRedeemer.java

```diff
@@ -0,0 +1,33 @@
+package io.blockfrost.sdk.api.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+/**
+ * ScriptRedeemer
+ */
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@ToString
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class ScriptRedeemer {
+    private String txHash;
+    private Integer txIndex;
+    private String purpose;
```

Comment:
Does enum make sense here (purpose)? As there are 4 possible values spend, mint, cert, reward

Reply:
I named it `ValidationPurpose` I am not sure of the naming. If you have a better name, feel free to advise.

Reply:
Sounds good to me